fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -160,12 +164,22 @@ public List<Column> listPartitionColumns(String databaseName, String tableName,
     public List<PartitionInfo> getPartitions(Table table, List<String> partitionNames) {
         try (Connection connection = getConnection()) {
             List<Partition> partitions = schemaResolver.getPartitions(connection, table);
+            String maxInt = IntLiteral.createMaxValue(Type.INT).getStringValue();
+            String maxDate = DateLiteral.createMaxValue(Type.DATE).getStringValue();
+
             ImmutableList.Builder<PartitionInfo> list = ImmutableList.builder();
-            if (partitions.size() > 0) {
+            if (!partitions.isEmpty()) {
```

Comment:
if (partitions.isEmpty()) {
return return Lists.newArrayList();
}
for (Partition partition : partitions) {
xxxxx;
}
code like this will be more simple

Reply:
done